src/BSON/UTCDateTime.c

```diff
@@ -509,32 +491,28 @@ static HashTable* php_phongo_utcdatetime_get_properties_hash(zval* object, bool
 {
 	php_phongo_utcdatetime_t* intern;
 	HashTable*                props;
-	char                      s_milliseconds[24];
-	int                       s_milliseconds_len;
 
 	intern = Z_UTCDATETIME_OBJ_P(object);
 
-	PHONGO_GET_PROPERTY_HASH_INIT_PROPS(is_debug, intern, props, 2);
+	PHONGO_GET_PROPERTY_HASH_INIT_PROPS(is_debug, intern, props, 1);
```

Comment:
Is this just cleaning up an extra allocation, since we know there's only one property to report?

Reply:
That’s correct.